setup.py

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env python
+
+from setuptools import setup
```

Comment:
doesn't `requirements.txt` need `setuptools`?

Reply:
No, this is needed only for building the distribution package, not for using the plugin.